gamedev-iitk/tower-defense
# Dynamic NavMesh

**Description:** Adding Dynamic Navmesh

**Steps to test:**
1.Move around the tower, and check the navmesh changes
2.Upgrade the towers and repeat step 1

**Outstanding:**
1. Size of the navmesh obstacle should be changed to a preferred value. I have given the radius and height values, you can use them as default if you want.
